aspnetcore/toc.yml

```diff
@@ -1459,12 +1459,14 @@ items:
           - name: Manage JWTs in development
             displayName: jwt, JSON WEB Token
             uid: security/authentication/jwt
-          - name: Azure Active Directory
+          - name: Azure
             items:
               - name: Overview
                 uid: security/authentication/azure-active-directory/index
-              - name: Integrate Azure AD into a web app
-                href: /azure/active-directory/develop/quickstart-v2-aspnet-core-webapp
+              - name: Secure ASP.NET Core apps with Microsoft Identity Web
+                href: /azure/active-directory/develop/
```

Comment:
This seems to be misnamed. Its link text says "Microsoft Identity Web", but the URL goes to [Microsoft Identity Platform](https://learn.microsoft.com/en-us/azure/active-directory/develop/). We should probably change the link text, or change the URL to something like https://github.com/AzureAD/microsoft-identity-web#readme
Either way I'm not sure if it should be under a TOC node named **Azure.**

cc @Rick-Anderson

Reply:
Hi @tdykstra yes, thanks

Microsoft.Identity..Web is the Nuget package (also best docs, see wiki) for integrating ASP.NET Core into the following three Azure platforms:
- Microsoft Entra ID
- Microsoft Entra External ID for customers
- Azure AD B2C

Possible link: https://github.com/AzureAD/microsoft-identity-web/wiki

I renamed the link to Microsoft identity platform and the menu to Microsoft Entra ID so it is as it was with the new product name and like it was before.

Good?

Reply:
Also some of the links in the samples are to other products within the Azure family. This is why Azure would be a better name, otherwise you need 4 different product links

Opinion?

Reply:
That makes sense, but maybe we should also add a node that links to the Microsoft Identity Web wiki.

Reply:
Should I add the link in the toc?